res/provider/ui/resolver/css/resolver.css

```diff
@@ -0,0 +1,146 @@
+@font-face {
+    font-family: Akkurat;
+    src: url(../assets/Akkurat.ttf);
+}
+
+:root {
+    /*
+        The standard left padding which everything is pushed over by.
+    */
+    --left-padding: 33px;
```

Comment:
Can this padding be on the container, rather than each of the components?

Reply:
No, the bars for each app need to be 100% width (of the window), otherwise you will see the borders being cut at the beginning of the padding.

Reply:
As discussed, no it cant as it will distort the full width elements